server/jobs/util-worker.js

```diff
@@ -2,12 +2,20 @@ const { parentPort, workerData } = require("worker_threads");
 const Database = require("../database");
 const path = require("path");
 
+/**
+ * Send message to parent process
```

Comment:
```suggestion
 * Send message to parent process for logging
 * since worker_thread does not have access to stdout, this is used instead of console.log()
```